code-gen/parse-csl/src/Csl/Gen/Pureness.hs

```diff
@@ -0,0 +1,147 @@
+module Csl.Gen.Pureness (
```

Comment:
I suggest we rename the type to `FunPurity` and move it to `Csl.Gen.Types.FunPurity`. 'Purity' should be a more conventional term in this context.

Reply:
Addressed in 275a386.